nrf_modem/doc/sockets/socket_options_func.rst

```diff
@@ -59,6 +59,8 @@ The following table shows all socket options supported by the Modem library.
 +-----------------+---------------------------------+------------------------+------------+--------------------------------------------------------------------------------------------+
 | NRF_SOL_SOCKET  | NRF_SO_EXCEPTIONAL_DATA         | ``int``                | get/set    | Send data on socket as part of exceptional event.                                          |
 +-----------------+---------------------------------+------------------------+------------+--------------------------------------------------------------------------------------------+
+| NRF_SOL_SOCKET  | NRF_SO_KEEPOPEN                 | ``int``                | get/set    | Keep the socket open when its PDN connection is lost, or the device is put in flight mode. |
```

Comment:
Remind me again, do we get the PDN down event in case of CFUN=4? If we lose the PDN connection in flight mode, maybe we can use flight mode as an example e.g. when the device is put in flight mode or out of reach from the base station. What about CFUN=20 (deactivate LTE)?

Reply:
Basically the application can `CFUN=4` and keep the socket open . It is not the case with `CFUN=0`.
`CFUN=20` seems to work the same as `CFUN=4`.